app/controllers/sufia/admin/permission_templates_controller.rb

```diff
@@ -21,7 +21,7 @@ def load_template_for_admin_set
 
         def update_params
           params.require(:sufia_permission_template)
-                .permit(:visibility, access_grants_attributes: [:access, :agent_id, :agent_type, :id])
+                .permit(:visibility, :workflow_name, access_grants_attributes: [:access, :agent_id, :agent_type, :id, :workflow_name])
```

Comment:
@jcoyne - I was just extending the existing update_params, I didn't see this in adminSetController. Should I be doing something instead of what I am at line 48 of adminSetController?